parser.py

```diff
@@ -0,0 +1,34 @@
+import bs4
+import requests
+
+# вставить адрес с сайта лостфильм.тв, раздел гид по сериям
+httpAddress = 'https://www.lostfilm.tv/series/Rome/seasons'
+
+# название сериала из адреса
+seriesName = ((str(httpAddress)).split('/')[-2])
+if '_' in seriesName:
+    newName = seriesName.split('_')
```

Comment:
Лучше использовать метод `replace()`
```suggestion
    seriesName = seriesName.replace('_', ' ')
```